prow/jobs/control-plane/tests/e2e/provisioning/e2e-provisioning-test-build.yaml

```diff
@@ -11,7 +11,7 @@ presubmits: # runs on PRs
         prow.k8s.io/pubsub.runID: "pull-e2e-provisioning-test-build"
         prow.k8s.io/pubsub.topic: "prowjobs"
         preset-sa-kyma-push-images: "true"
-      run_if_changed: '^tests/e2e/provisioning/|^scripts/'
+      run_if_changed: '^tests/e2e/provisioning/|^scripts/|^resources/kcp/charts/provisioner/'
```

Comment:
Please adjust templates of the jobs e.g. https://github.com/kyma-project/test-infra/blob/0e2a15014d9c1a3ea5a00073869aafb0e514f6e2/templates/data/control-plane-build.yaml#L374 and run make in the main folder.